scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1346,13 +1346,13 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
 
     If M is specified, solves ``A * x[i] = w[i] * M * x[i]``, the
     generalized eigenvalue problem for w[i] eigenvalues
-    with corresponding eigenvectors x[i]
+    with corresponding eigenvectors x[i].
 
     Parameters
     ----------
     A : An N x N matrix, array, sparse matrix, or LinearOperator representing
-        the operation A * x, where A is a real symmetric matrix
-        For buckling mode (see below) A must additionally be positive-definite
+        the operation A * x, where A is a real symmetric matrix.
```

Comment:
Asterisks have special meaning in ReST, so should wrap A*x into double backticks.

Reply:
Also it's better to use ``A @ b`` for matrix multiplication.